docs/wiki/ALZ-Policies.md

```diff
@@ -289,3 +289,63 @@ This management group is for subscriptions that will only be used for testing an
 | `Policy Definition Sets`  | **0**     |
 | `Policy Definitions`      | **0**     |
 </td></tr> </table>
+
+### Versioning
+
+Each policy definition and initiative contains a version in its metadata section:
+```json
+"metadata": {
+   "version": "1.0.0",
+   "category": "{categoryName}",
+   "source": "https://github.com/Azure/Enterprise-Scale/",
+   "alzCloudEnvironments": [
+      "AzureCloud",
+      "AzureChinaCloud",
+      "AzureUSGovernment"
+   ]
+}
+```
+
+This version is incremented according to the following rules (subject to change):
+   - **Major Version** (**1**.0.0)
+      - Policy Definitions
+         - Rule logic changes
+         - ifNotExists existence condition changes
+         - Major changes to the effect of the policy (i.e. adding a new resource to a deployment)
+      - Policy Set Definitions
+         - Addition or removal of a policy definition from the policy set
+   - **Minor Version** (1.**0**.0)
+      - Policy Definitions
+         - Changes to effect details that don't meet the major version criteria
+         - Adding new parameter allowed values
+         - Adding new parameters (with default values)
+         - Other minor changes to existing parameters
+      - Policy Set Definitions
+         - Adding new parameter allowed values
+         - Adding new parameters (with default values)
+         - Other minor changes to existing parameters
+   - **Patch Version** (1.0.**0**)
+      - Policy Definitions
+         - String changes (displayName, description, etc…)
+         - Other metadata changes
+      - Policy Set Definitions
+         - String changes (displayName, description, etc…)
+         - Other metadata changes
+   - **Suffix**
+      - Append "-preview" to the version if the policy is in a preview state  
+         - Example:  1.3.2-preview
+      - Append "-deprecated" to the version if the policy is in a deprecated state
+         - Example:  1.3.2-deprecated
+ 
+## Preview and deprecated policies
+
+This section aims to explain what it means when a built-in policy has a state of ‘preview’ or ‘deprecated’.  
+
+Policies can be in preview because a property (alias) referenced in the policy definition is in preview, or the policy is newly introduced and would like additional customer feedback. A policy may get deprecated when the property (alias) becomes deprecated & not supported in the resource type's latest API version, or when there is manual migration needed by customers due to a breaking change in a resource type's latest API version. 
+
+When a policy gets deprecated or gets out of preview, there is no impact on existing assignments. Existing assignments continue to work as-is. The policy is still evaluated & enforced like normal and continues to produce compliance results.  
+
+Here are the changes that occur when a policy gets deprecated: 
```

Comment:
Is it worth calling out that the policy `name` must not, and will not, be changed?

Also the fact that if a deprecated policy is being replaced by a new one representing the same end goal, it will need to have a different `name` to the original one to avoid potential conflicts or issues when upgrading?

Reply:
I'll include some detail on the names not changing, just the addition of what's on this point. 

I think the appended `deprecated` at the start of the name, alongside the 'builtin' type instead of Azure Landing Zones should be enough to say what's happening, even if they were to share the exact same name. Potentially including a renaming of a policy name if it's being deprecated could cause a little confusion rather than just seeing a policy with the same name and `deprecated` prefixed to it. Also as part of instructions, we say to migrate/delete the deprecated policy. Thoughts?

Reply:
We need to be clear on `name` vs. `displayName`... the changes above are to `displayName` and are non-breaking. Changes to `name` effectively make it a completely new policy in Azure so cannot be changed / duplicated without consequences.

Reply:
Understood, I'll get it in!

Reply:
Please can we update from `does not change` to `must not change`, and include reference re-using the name of a deprecated policy when creating new versions?

Reply:
amended